compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1588,7 +1588,7 @@ rustc_queries! {
         arena_cache
         desc { "resolving lifetimes" }
     }
-    query named_region_map(_: LocalDefId) ->
+    query named_region_map(_: hir::OwnerId) ->
```

Comment:
This requires to add an `impl DepNodeParams for OwnerId` which copies the one for `LocalDefId`.